### DISCUSSION TITLE
After putty upgrade everything is on chineese

### DISCUSSION THREAD

**Discussion by Mile-Lile** [linuxdeepin/developer-center#3881] [Troubleshooting | 故障排查]
After latest upgade my putty went from english to chineese version and now O cannot use it... It is just tottaly unreadable to me and I cannot revert it. I have respect to chineese tradition and culture but it is like I see ancient hieroglyphs...
![Screenshot_select-area_20230313191328](https://user-images.githubusercontent.com/12752816/224793644-6d07493a-3020-426c-a692-f7674207434a.png)

**Comment by BLumia** [comment]
cc @Zeno-sole @babyfengfjx
